Wireless.py

```diff
@@ -159,6 +163,9 @@ def _getStability(self, sender, receiver):
             (receiver.dotBotId, sender.dotBotId)
         }.intersection(self.lastStabilities.keys())
 
+        # convert set to list and sort, to maintain order for  reproducibility
+        linkInLastStabilities = list(sorted(list(linkInLastStabilities)))
```

Comment:
is the right-most `list(` needed? It looks like `linkInLastStabilities` is a list.
Also, please detail what this is so we can agree on a name for it